src/OSPSuite.SimModel/src/DESolver.cpp

```diff
@@ -476,6 +476,8 @@ namespace SimModelNative
 		for (i = 0; i<_parentSim->Observers().size(); i++)
 		{
 			Observer * observer = _parentSim->Observers()[i];
+			if (!observer->IsUsedInSimulation())
+				continue;
 
```

Comment:
ignore unused observers.
Actually I'm not sure if they shpuld be completely removed from the system or not.
Depends also a bit on the toolbox behaviour, which we didn't finally discussed yet (e.g. if the toolbox should be able to set an Observer as persistable after a simulation was initialized).
So for now all those Observers are kept, but ignored for all operations.